### ISSUE TITLE
Proguard 7.0 not (yet) deployed at maven central 

### ISSUE THREAD

**Issue by wumpz** [Guardsquare/proguard#54]
The highest version I found was a 6.2.2. Do I need to use another repository?

**Comment by EricSalemi** [comment]
Our intention is to release to Maven Central soon.

For the moment the `7.0.0` release is available on JCenter.

**Comment by TWiStErRob** [comment]
Long time has passed, I can't find 7.0.0 anywhere. I checked
https://bintray.com/guardsquare/proguard/net.sf.proguard%3Aproguard-gradle (not even 6.2.2)
https://mvnrepository.com/artifact/net.sf.proguard/proguard-gradle (6.3 beta is latest)

Does this have to do with https://github.com/Guardsquare/proguard/releases/tag/v7.0.0
> PGD-12 | Fixed building gradle plugin.

Note: we use these in `buildSrc`:
```gradle
repositories {
    google()
    jcenter()
    gradlePluginPortal()
}
dependencies {
    implementation("net.sf.proguard:proguard-gradle:6.2.2")
}
```

**Comment by mrjameshamilton** [comment]
Hi @TWiStErRob , as of 7.0.0 the group ID is now `com.guardsquare`.

```
implementation("com.guardsquare.proguard-gradle:7.0.0")
```

**Comment by TWiStErRob** [comment]
Ahaa :) Thanks. Can you please put that in the release notes for 7.0.0?

https://www.guardsquare.com/en/products/proguard/manual/versions
https://github.com/Guardsquare/proguard/releases/tag/v7.0.0

**Comment by TheDGOfficial** [comment]
Now that JCenter is closing, will ProGuard be uploaded to Maven-Central? I tried Jitpack as an alternative but it ended up with the error "Cannot perform signing task ':signParentPublication' because it has no configured signatory"

**Comment by mrjameshamilton** [comment]
Hi @TheDGOfficial ! We're aware of the closure of jcenter and we're working out a solution but we don't have an answer yet.

**Comment by ArcticLampyrid** [comment]
Consider publishing to Gradle Plugin Portal?
It's the default repo for `plugins` dsl.

https://plugins.gradle.org/

**Comment by vlad-lubenskyi** [comment]
I was able to resolve the plug-in using `https://plugins.gradle.org/m2/` repo. However, if you look at the portal or browse the repository in your browser, you won't find it.

Is it safe to start using this repo?

**Comment by TWiStErRob** [comment]
@vlad-lubenskyi no, that's still jcenter(), open this url for proof: https://plugins.gradle.org/m2/proguard-gradle/7.0.0

**Comment by TWiStErRob** [comment]
And documentation: "How do you know if you’re using JCenter?" on https://blog.gradle.org/jcenter-shutdown

**Comment by s17t** [comment]
Any chances to see the beta published on maven central too ?

**Comment by harpocrates** [comment]
This is becoming really pressing. JCenter is starting to return 403's, and the May 1st 2021 deadline is less than a week away...

**Comment by EricSalemi** [comment]
Publishing infrastructure for Sonatype landed in #152. 

@s17t the next beta will be published on Maven Central.

**Comment by mrjameshamilton** [comment]
Good news - ProGuard is now available on Maven Central https://search.maven.org/search?q=g:com.guardsquare

**Comment by harpocrates** [comment]
Could the 7.0.0 and 7.0.1 versions of ProGuard libraries also be copied over to Maven Central? For instance, it seems like only version 7.1.0-beta3 of[`proguard-base`](https://mvnrepository.com/artifact/com.guardsquare/proguard-base) is on maven central.

We can stomach upgrading to a beta version of the libraries, but I'm worried that if 7.0.* artifacts are never published, our old commits (which reference 7.0.0 artifacts) will simply no longer work.

**Comment by guillerodriguez** [comment]
> Could the 7.0.0 and 7.0.1 versions of ProGuard libraries also be copied over to Maven Central? For instance, it seems like only version 7.1.0-beta3 of[`proguard-base`](https://mvnrepository.com/artifact/com.guardsquare/proguard-base) is on maven central.
> 
> We can stomach upgrading to a beta version of the libraries, but I'm worried that if 7.0.* artifacts are never published, our old commits (which reference 7.0.0 artifacts) will simply no longer work.

Yes please. Exactly the same problem here.
